polkadot.json

```diff
@@ -83,37 +83,37 @@
       },
       {
         "id": 11,
-        "imageUrl": "https://i.imgur.com/YQNFrNk.png",
-        "title": "Kusama - Polkadot's wild cousin",
-        "details": "When you're developing a website, you usually have three stages: development, staging, and production. But in the blockchain world, it's a bit different. There's usually just development and production - that is, until you meet Kusama! Kusama is like Polkadot's adventurous cousin. It lets developers try out their parachains under real world conditions in a “canary network” before they're released on the main Polkadot network."
+        "imageUrl": "https://i.imgur.com/mli9QVH.png",
+        "title": "Scaling itself",
+        "details": "Polkadot is more than just a scaling solution for other applications and blockchains, it scales itself! Features on Polkadot are also parallelized and sharded allowing cheap and efficient access for all users of Polkadot. That is why when using Polkadot, you can expect fast speeds, low fees, and high security."
       },
       {
         "id": 12,
         "imageUrl": "https://i.imgur.com/SHmsZPe.png",
-        "title": "Canary network - explained!",
-        "details": "A canary network is a live network that allows developers to test their blockchain under real-world conditions before launching it on the mainnet. It’s different from a testnet because it carries real value and transactions, providing a more authentic environment for testing. Think of Kusama like a fun and exciting adventure, where developers can take risks and test their limits. This helps create a stronger and more secure Polkadot ecosystem in the end!"
+        "title": "Play on Polkadot",
+        "details": "Polkadot offers a suite of features available to all $DOT holders. You can create and distribution new assets and NFTs, participate in decentralized governance, secure the network and earn rewards via staking, securely bridge to other blockchains, and soon even deploy smart contracts directly to the network."
```

Comment:
"soon even deploy smart contracts to the network" might sound a bit strange without context on why we refrained from it, to begin with.

Reply:
fair. perhaps just wait until we have it, and then talk directly about it

Reply:
or we can say we have it and reference moonbeam etc